tests/perf/pubsub_subscribe_http/pubsub_subscribe_http_default_component_test_config.yaml

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
Could rename the file to `test_kafka.yaml` or something like that

Shorter names allow distinguishing the file more easily when you look at them in your IDE!